anime_downloader/sites/helpers/selescrape.py

```diff
@@ -0,0 +1,191 @@
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from anime_downloader.const import get_random_header
+from selenium.webdriver.common.by import By
+from urllib.parse import urlencode
+from urllib.parse import urlsplit
+from selenium import webdriver
+from bs4 import BeautifulSoup
+from logging import exception
+from sys import platform
+import requests
+import os
+import click
+import time
+import json
+
+
+def get_data_dir(): #gets the folder directory selescrape will store data, such as cookies or browser extensions and logs.
```

Comment:
This is not how you document functions. Look at other functions in repo or google for python docstring conventions.

Basically use `'''` comments

Reply:
ok, i will use them from now on